ui/src/data-services/models/session-details.ts

```diff
@@ -16,19 +16,21 @@ interface SummaryData {
 }
 
 export class SessionDetails extends Session {
-  private readonly _firstCapture: Capture
+  private readonly _firstCapture?: Capture
 
   public constructor(event: ServerEventDetails) {
     super(event)
 
-    this._firstCapture = new Capture(event.first_capture)
+    if (event.first_capture) {
```

Comment:
Hmm I think best if not, because I want to avoid create a new instance of Capture each time the getter is called. That is why I'm doing it in the constructor :)

Reply:
Got it, thanks for the explanation!